packages/flutter/lib/src/rendering/layer.dart

```diff
@@ -193,12 +194,17 @@ class PerformanceOverlayLayer extends Layer {
   /// that aid it in making better decisions about caching.
   final bool checkerboardRasterCacheImages;
 
+  /// Whether the compositor should checkerboard layers that are rendered to offscreen
+  /// bitmaps. This can be useful for debugging rendering performance.
+  final bool checkerboardOffscreenLayers;
```

Comment:
Can you include a paragraph that discusses how this differs from `checkerboardRasterCacheImages`?